src/test/java/org/cactoos/list/ListNoNullsTest.java

```diff
@@ -101,4 +101,34 @@ public void removeThrowsErrorIfValueNull() {
             list
         ).remove(0);
     }
+
+    @Test
+    public void iteratorThrowsErrorWithRemoveOperation() {
+        this.exception.expect(UnsupportedOperationException.class);
+        final ArrayList<Integer> list = new ArrayList<>(1);
+        list.add(1);
+        new ListNoNulls<>(
+            list
+        ).removeIf(value -> value == 1);
```

Comment:
@DanielPalafox the method intended for test `Iterator.remove`, not `List.removeIf`